keyboards/rainkeebs/trailmix/info.json

```diff
@@ -0,0 +1,117 @@
+{
+    "manufacturer": "rainkeebs",
```

Comment:
considering I don't have any previous experience with C and qmk flashing, I put here everything that came out of `qmk info -f json -kb rainkeebs/trailmix`. Happy to slim it down if necessary.